library/nxos_switchport

```diff
@@ -72,17 +72,31 @@ options:
         aliases: []
     trunk_vlans:
         description:
-            - if mode=trunk, used as the vlan range to carry over trunk
```

Comment:
Logic is good, but can you re-word?  Maybe something like "vlan(s) to add or remove to a trunk when mode=trunk.  Adding or removing is based on value of state parameter"

Note: need to remove all colons from the docstring.  Let's keep on one indent level too.  Can't break the creation of Ansible docs (that reads this docstring in automatically).